sympy/parsing/mathematica.py

```diff
@@ -8,13 +8,20 @@
 
 
 def mathematica(s, additional_translations=None):
-    '''Users can add their own translation dictionary
-    # Example
-    In [1]: mathematica('Log3[9]', {'Log3[x]':'log(x,3)'})
-    Out[1]: 2
-    In [2]: mathematica('F[7,5,3]', {'F[*x]':'Max(*x)*Min(*x)'})
-    Out[2]: 21
-    variable-length argument needs '*' character '''
+    '''
+    Users can add their own translation dictionary.
+    variable-length argument needs '*' character.
+    
+    Examples
+    ========
```

Comment:
This looks good, but I think you ran python trace command which created these .cover files and then added in the commit accidentally. You have to remove these files!

Reply:
It is not sufficient to remove these files in a new commit. They should not be there in the history either.

I suggest to start this PR again from master.

Reply:
What is .cover? Should we add this to the .gitignore?

Reply:
I looked at the documentation for the `trace` command. The command basically traces the program executions and produces the Code coverage in the `.cover` files.

The files list which statements were executed how many times and which statements were not executed during program execution. 

Although the developers should care to not add the files if they use the trace command, it might be a good idea to add the .cover files to .gitignore, just in case.

Reply:
I was trying to identify some bottlenecks in `lagrange.py` in the mechanics package and was trying some ways to do that and found the `trace` command as one of them. So basically any module/function called by the `lagrange.py` file generated a coverage report and was stored in the same repo. I forgot about this and could not recognise where the `.cover` files came from and mistakenly committed them